staging/src/k8s.io/apimachinery/pkg/util/runtime/runtime.go

```diff
@@ -79,12 +85,26 @@ func getCallers(r interface{}) string {
 	return callers
 }
 
+// these constants determine the default behavior of dedupingErrorHandler
+const (
+	// cacheSize determines how many "unique" errors to track (see errKey)
+	cacheSize = 1000
+	// errorDepth determines how many callers need to be skipped to find the stack of HandleError's caller
+	// HandleError -> ErrorHandlers iteration -> dedupingErrorHandler.handleErr -> dedupingErrorHandler.logError
+	errorDepth = 4
+	// delta determines how long we can go in between logging the same error that we have seen multiple times
+	// it mainly exists to handle the extreme ends of the error spectrum:
+	// 1. a component that sends the same error very infrequently (we want to log these so we do not miss them)
+	// 2. a component that sends the same error as fast as it can (we want to suppress most of these, but still see it every so often)
+	delta = time.Hour
```

Comment:
I would have chosen something closer to 5 minutes.

Reply:
Yeah

Reply:
OK